dao/seeds/application_types.yml

```
@@ -69,3 +69,11 @@
   supported_authentication_types:
     satellite:
     - receptor_node
+"/insights/platform/provisioning-service":
+  display_name: Provisioning service
```

I'd rather go with Provisioning only, but unimportat ATM :)
```suggestion
  display_name: Provisioning
```